pulsar/consumer_test.go

```diff
@@ -280,7 +280,7 @@ func TestConsumerKeyShared(t *testing.T) {
 	defer producer.Close()
 
 	ctx := context.Background()
-	for i := 0; i < 10; i++ {
+	for i := 0; i < 100; i++ {
```

Comment:
What's the purpose of changing this from 10 -> 100?

Reply:
Just to ensure that every consumer receives a message